### DISCUSSION TITLE
vmalert?

### DISCUSSION THREAD

**Discussion by ananthb** [VictoriaMetrics-Community/homeassistant-addon-victoriametrics#55] [Ideas]
Any plans for a vmalert addon along similar lines?

If I fork this and make one for vmalert is there interest in adding it to the `VictoriaMetrics-Community` GitHub org?
I can maintain it as well.

**Comment by denisgolius** [comment]
> Any plans for a vmalert addon along similar lines?
> 
> If I fork this and make one for vmalert is there interest in adding it to the `VictoriaMetrics-Community` GitHub org? I can maintain it as well.

Could you describe your case?

**Reply by ananthb** [reply]
I'm running victoriametrics as an add-on on home assistant. I would like to run vmalert next to it now.

**Reply by denisgolius** [reply]
> I'm running victoriametrics as an add-on on home assistant. I would like to run vmalert next to it now.

You may also run vmalert as a Docker container on your HA. If you want to create an add-on for vmalert and support it in the community, go ahead!

PS: I believe that using VictoriaMetrics as long-term storage at HA is not a good idea. I mean, HA is for IoT, but it is not something that can replace a VPS with 1 CPU and 2 RAM for running VM Single and store your monitoring data there because of flexibility.

**Reply by ananthb** [reply]
> You may also run vmalert as a Docker container on your HA.

I'm using the official OS and the only way to run containers on this afaik is addons.

> ...HA is for IoT, but it is not something that can replace a VPS with 1 CPU and 2 RAM for running VM Single...

I'm also running the OS on some beefy hardware with NVMe disks and plenty of CPU and RAM. I'd much rather prefer a locally hosted victoriametrics and vmalert!

> If you want to create an add-on for vmalert and support it in the community, go ahead!

Sounds good!

**Reply by denisgolius** [reply]
сс @vrutkovs
